pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                               label=('rotation matrix value 11 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat11_cmd + '?',
+                               set_cmd=srotmat11_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            '''
+            TV mode parameters
+            '''
+            sintavg_cmd = 'qutech:tvmode{}:naverages'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_naverages'.format(ch_pair),
+                               units='#',
+                               label=('the number of integration averages' +
+                                      'of ch_pair {} '.format(ch_pair)),
+                               get_cmd=sintavg_cmd + '?',
+                               set_cmd=sintavg_cmd + ' {}',
+                               vals=vals.Numbers(1, 1024)
+                               )
+            stvseg_cmd = 'qutech:tvmode{}:nsegments'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_nsegments'.format(ch_pair),
+                               units='#',
+                               label=('the number of TV segments of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stvseg_cmd + '?',
+                               set_cmd=stvseg_cmd + ' {}',
+                               vals=vals.Numbers(1, 256)
+                               )
+            stven_cmd = 'qutech:tvmode{}:enable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_enable'.format(ch_pair),
+                               label=('enable tv mode' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stven_cmd + '?',
+                               set_cmd=stven_cmd
+                               # vals=vals.Numbers(1,4096)
+                               )
+            stvdis_cmd = 'qutech:tvmode{}:disable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_disable'.format(ch_pair),
+                               label=('disable tv mode' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=stven_cmd + '?',
+                               set_cmd=stvdis_cmd,
+                               # vals=vals.Bool()
+                               )
+            self.add_parameter('ch_pair{}_tvmode_data'.format(ch_pair),
+                               label=('get TV data ch {} '.format(ch_pair)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getTVdata, ch_pair)
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+            scaladc_cmd = 'qutech:caladc{}'.format(ch_pair)
+            self.add_parameter('ch_pair{}_cal_adc'.format(ch_pair),
```

Comment:
From what I can see, this is not a parameter (it's not getable) but rather a function. Calibrate ADC. 

I don't know what it does so maybe it could be a parameter but it doesn't look like it.